experiment/measurer/measure_worker.py

```diff
@@ -108,7 +108,8 @@ def _get_previous_cycle_state(self) -> list:
             self._get_bucket_cycle_state_file_path(self.cycle - 1))
 
         return json.loads(
-            gsutil.cat(previous_state_file_bucket_path, expect_zero=False))
+            filestore_utils.cat(previous_state_file_bucket_path,
+                                must_exist=False).output)
```

Comment:
We can just return output from cat function itself.

Reply:
I want it to have the same interface (e.g. returns a ProcessResult) as the rest of the functions in the module.